proto/buf/alpha/registry/v1alpha1/repository_track_commit.proto

```diff
@@ -21,21 +21,14 @@ import "google/protobuf/timestamp.proto";
 // RepositoryTrackCommit is the existance of a RepositoryCommit on a RepositoryTrack. Currently its only purpose is
 // for querying whether a RepositoryCommit is on a RepositoryTrack and determining it's sequence id.
 message RepositoryTrackCommit {
-  // We reserve field number '3' for id, but RepositoryTrackCommits have no id.
-  // string id = 1;
```

Comment:
I investigated this - it looks like a bug with `protocompile`. The leading detached comment for the first field in a message isn't attached to any of the nodes on the `*ast.FileNode`. I'll see if I can fix this.

Reply:
I think this might be parsed as the _trailing_ comment for message `RepositoryTrackCommit`, not a leading or detached comment.

I just did a quick test with this file, to round-trip the file -- parse to AST and then print the AST back out. And this comment is in there. So it's in the AST somewhere. Perhaps it's just not where your formatter expects it?

Reply:
Thanks for the heads up, you're probably right. I tested this earlier by using the `ast.SimpleVisitor` and printed the leading/trailing comments attached to every node. I might have made a mistake somewhere tho - I'll figure it out!

Reply:
Discussed offline - it turns out that it's because the comment is attached to the `{` node - not the message itself. This is also true for other composite nodes like `enum`, `extend`, etc. I'll revisit this and make sure to handle these cases.